src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -1,11 +1,90 @@
 package frc.team2412.robot.subsystem;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonFXFeedbackDevice;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
-
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ShooterSubsystem extends SubsystemBase {
-    public ShooterSubsystem(WPI_TalonFX flywheelMotor1, WPI_TalonFX flywheelMotor2, WPI_TalonFX turretMotor, WPI_TalonFX hoodMotor) {
+    private final WPI_TalonFX flywheelMotor1;
+    private final WPI_TalonFX flywheelMotor2;
+    private final WPI_TalonFX turretMotor;
+    private final WPI_TalonFX hoodMotor;
+
+    public static final double MAX_FORWARD = 0.1;
+    public static final double MAX_REVERSE = -0.1;
+    public static final double STOP_MOTOR = 0;
+    public static final double DEGREES_TO_ENCODER_TICKS = 2048 / 360; // 2048 ticks per 360 degrees
+    public static final double MIN_TURRET_ANGLE = 0;
+    public static final double MAX_TURRET_ANGLE = 180; // Placeholder, maybe need to update
+
+    public ShooterSubsystem(WPI_TalonFX flywheelMotor1, WPI_TalonFX flywheelMotor2, WPI_TalonFX turretMotor,
+            WPI_TalonFX hoodMotor) {
+        this.flywheelMotor1 = flywheelMotor1;
+        this.flywheelMotor2 = flywheelMotor2;
+        this.flywheelMotor2.setInverted(true);
+        this.turretMotor = turretMotor;
+        this.hoodMotor = hoodMotor;
+        // Make sure hood is using builtin encoder, may be unneccessary
+        this.hoodMotor.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 0);
+    }
+
+    public void hoodMotorExtend() {
+        hoodMotor.set(MAX_FORWARD);
+    }
+
+    public void hoodMotorRetract() {
+        hoodMotor.set(MAX_REVERSE);
+    }
```

Comment:
Not sure if these would be useful for the drivers, as I'd expect manual adjustment to just move the hood a small amount for each button press. Perhaps have these get the current hood target position (new class variable), then add (or subtract) a small amount, and set the hood motor to a new target position.

Reply:
(We decided to just replace the extend and retract methods with a method which sets the angle, without any nudging methods since we didn't think they'd be useful)